modules/@apostrophecms/admin-bar/index.js

```diff
@@ -84,8 +85,10 @@ module.exports = {
 
       add(name, label, permission, options) {
         let index;
+
         const item = {
-          name: name,
+          name: name.indexOf(':') === -1 ? name : name.split(':')[0],
```

Comment:
Maybe the argument in `add` should be `action` instead of `name`?

Reply:
`action` could get confused with the action URL of a module. I kinda like name.

Reply:
But I'm flexible on it.

Reply:
yeah, that's true. The main case for it here is that it's going to get passed down into AposContextMenu components, which emit the items' `action` as well. There is potential for confusion, though.